README.rst

```diff
@@ -218,11 +225,77 @@ Find PUDL useful? Want to help make it better? There are lots of ways to help!
   `Github issue tracker <https://github.com/catalyst-cooperative/pudl/issues>`__.
 * Feel free to fork the project and make a pull request with new code, better
   documentation, or example notebooks.
-* `Make a recurring financial contribution <https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=PZBZDFNKBJW5E&source=url>`__
-  to support our work liberating public energy data.
 * `Hire us to do some custom analysis <https://catalyst.coop/hire-catalyst/>`__ and
   allow us to integrate the resulting code into PUDL.
 
+PUDL Sustainers
```

Comment:
Formatting options here are limited and this isn't beautiful, but it gets the necessary information across.